docs/CONTRIBUTING.md

```diff
@@ -28,8 +28,8 @@
 
 ## The Target Version
 
-* The standard library signatures targets Ruby 2.7 for now.
-* The library code targets Ruby 2.6, 2.7, and 3.0.
+* The standard library signatures targets Ruby 3.0 for now.
+* The library code targets Ruby 3.0.
```

Comment:
The signature target is Ruby 3.0 now. But the library code targets are 2.7 and 3.0.
I'm sorry for the confusion. 🙇‍♂️

Reply:
Thank you for letting me know! 😄 
So reverted some changes in https://github.com/ruby/rbs/pull/633/commits/bdc86b12ce04b5f29c80376c7b6caec3bf08a029 🚮 

By the way, should we update gemspec too? https://github.com/ruby/rbs/blob/3bf688260bb4562335fdd8af0cd33e230f09db76/rbs.gemspec#L37

Reply:
Leave it as is. (I'm not sure if rejecting all of the `2.6` users makes sense...)

Reply:
Sure!